native/tools/swig-config/scene.i

```diff
@@ -284,7 +285,7 @@ using namespace cc;
 %attribute_writeonly(cc::Node, Mat4&, matrix, setMatrix);
 %attribute(cc::Node, uint32_t, hasChangedFlags, getChangedFlags, setChangedFlags);
 %attribute(cc::Node, bool, _persistNode, isPersistNode, setPersistNode);
-%attribute(cc::Node, cc::MobilityMode, _mobility, getMobility, setMobility);
+%attribute(cc::Node, cc::MobilityMode, mobility, getMobility, setMobility);
```

Comment:
Why change it? I don't see any modification of the implementation.

Reply:
https://github.com/cocos/cocos-engine/blob/develop/cocos/scene-graph/node.ts#L1679

To keep the same logic as web.

_mobilty is a public property that will be automatically bound, no need to use an attribute directive to declare it. 

The old code will  not bind `mobility` property to JS.